styleguide-app/Examples/AssignmentIcon.elm

```diff
@@ -53,5 +53,20 @@ example =
                 [ ( "startPrimary", AssignmentIcon.startPrimary )
                 , ( "startSecondary", AssignmentIcon.startSecondary )
                 ]
+            , IconExamples.view "Activities"
+                [ ( "assessment", AssignmentIcon.assessment )
+                , ( "practice2", AssignmentIcon.practice2 )
+                , ( "standards", AssignmentIcon.standards )
+                , ( "writing", AssignmentIcon.writing )
+                ]
+            , IconExamples.view "Practice & Assessment"
+                [ ( "assignArrow", AssignmentIcon.assignArrow )
+                , ( "home", AssignmentIcon.home )
+                , ( "passageQuiz", AssignmentIcon.passageQuiz )
```

Comment:
Why do passageQuiz, planningDiagnostic, unitDiagnostic, and quiz2 not belong in the quiz engine section up above?

That makes me wonder if `diagnostic` belongs in the "Activities" section 🤔